src/type_definitions/react_virtualized.d.ts

```diff
@@ -0,0 +1,22 @@
+/*
```

Comment:
This has to be in the "Kibana" part of the project otherwise there is an TSLint failure because of EUI. If you put this under our project it doesn't fix the typing issue.

Reply:
:thinking: deleting this file does not seem to break anything for me. is it not needed anymore after the EUI downgrade?